.ideainspect

```diff
@@ -1,12 +1,6 @@
 # Levels to look for. Default: WARNING,ERROR
 levels: WARNING,ERROR,INFO
 
-# Inspection result files to skip. For example "TodoComment" or "TodoComment.xml".
-skip: TodoComment,Annotator
-
-# Ignore issues affecting source files matching given regex. Example ".*/generated/.*".
-skipfile: .*/generated/.*,src/main/Foo.java
```

Comment:
For backward compatibility we should keep this option and mark it deprecated and advise user to prefer the scope for performance & clarity reasons. Never break a running system.